sdk/src/test/java/io/dapr/client/DaprHttpTest.java

```diff
@@ -139,7 +144,7 @@ public void invokeGetMethod() throws IOException {
     mockInterceptor.addRule()
       .get("http://127.0.0.1:3500/v1.0/get")
       .respond(serializer.serialize(EXPECTED_RESULT));
-    DaprHttp daprHttp = new DaprHttp(Properties.SIDECAR_IP.get(), 3500, okHttpClient);
+    DaprHttp daprHttp = new DaprHttp(Properties.SIDECAR_IP.get(), 3500, okHttpClient, DEFAULT_PARSER);
```

Comment:
Use an overloaded version of the constructor instead. So, there will not be changes needed everywhere.